### PR TITLE
use `generate-cert.sh` to generate caBundle and tls.* files

### PR DESCRIPTION
**Pull request by tpiperatgod**
I think we can remove the dependency on the cert-manager. We can use `generate-cert.sh` to generate the information we need for the certification process.

This requires us to go into the `hack` directory and manually execute the `bash generate-cert.sh` command before `make manifests`. This action will automatically replace the certification information in the relevant files.

Signed-off-by: laminar <fangtian@kubesphere.io>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,6 +55,7 @@ help: ## Display this help.
 
 ##@ Development
 
+# Before executing make manifests, you need to go to the `hack` directory and execute the `generate-cert.sh` script.
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 	kubectl kustomize config/default | sed -e '/creationTimestamp: null/d' | sed -e 's/openfunction-system/openfunction/g' | sed -e 's/openfunction\:latest/openfunction\:$(VERSION)/g' | sed -e 's/app.kubernetes.io\/version\: latest/app.kubernetes.io\/version\: $(VERSION)/g' > config/bundle.yaml
```
Comment (benjaminhuo) [inline Makefile:61]: Can we add the call to `generate-cert.sh` before `kubectl kustomize config/default | ...` directly?

File: config/bundle.yaml [hunk 2]
```diff
@@ -30408,55 +30408,6 @@ status:
   conditions: []
   storedVersions: []
 ---
-apiVersion: admissionregistration.k8s.io/v1
```
Comment (benjaminhuo) [inline config/bundle.yaml:30411]: We should also change Makefile from

`manifests: controller-gen`

to

`manifests: verify controller-gen`

This way everytime when you run make manifests, it will first format the code in a consistent way to avoid unnecessary changes everytime

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (tpiperatgod) [reply Makefile:61]: This is only possible if "verify" is adjusted to before "manifests", but then does "verify" lose its meaning?

File: Makefile [hunk 3]
```diff
@@ -55,7 +55,8 @@ help: ## Display this help.
 
 ##@ Development
 
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+# Before executing make manifests, you need to go to the `hack` directory and execute the `generate-cert.sh` script.
```
[the remark this answers is not in the document]
Comment (tpiperatgod) [reply Makefile]: removed it..

**Review by benjaminhuo — APPROVED** [review]
